lib/ex_doc/formatter/html.ex

```diff
@@ -212,7 +213,7 @@ defmodule ExDoc.Formatter.HTML do
 
   defp build_api_reference(nodes_map, config) do
     api_reference = Templates.api_reference_template(config, nodes_map)
-    %{id: "api-reference", title: "API Reference", group: "", content: api_reference}
+    %{id: "api-reference", title: "<%= translate(config, "API Reference") %>", group: "", content: api_reference}
```

Comment:
Sorry, it doesn't compile, as it was not a template.

```suggestion
    %{id: "api-reference", title: translate(config, "API Reference"), group: "", content: api_reference}
```

We allso need to add this in line no. 6
` 
  import ExDoc.Translation
`